lib/client.js

```diff
@@ -42,6 +42,8 @@ var Client = module.exports = function(config) {
     this.brokers = new Brokers(this);
     this.topics = new Topics(this);
     this.consumers = new Consumers(this);
+
+    if (config.timeout) this.config.timeout = config.timeout;
```

Comment:
Agree on both points. I think I did that way just to limit the amount of code change.